tests/integration/kubernetes/confidential_kbs.sh

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2024 Red Hat
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Provides a library to deal with the CoCo KBS
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+kubernetes_dir="$(dirname "$(readlink -f "$0")")"
+# shellcheck disable=1091
+source "${kubernetes_dir}/../../gha-run-k8s-common.sh"
+
+# Where the kbs sources will be cloned
+readonly COCO_KBS_DIR="/tmp/kbs"
+
+# Delete the kbs on Kubernetes
+#
+# Note: assume the kbs sources were cloned to $COCO_KBS_DIR
+#
+function kbs_k8s_delete() {
+	pushd "$COCO_KBS_DIR"
+	kubectl delete -k kbs/config/kubernetes/overlays
+	popd
+}
+
+# Deploy the kbs on Kubernetes
+#
+function kbs_k8s_deploy() {
+	local image
+	local image_tag
+	local ns="coco-tenant"
+	local repo
+	local kbs_ip
+	local kbs_port
+	local version
+
+	# yq is needed by get_from_kata_deps
+	ensure_yq
+
+	# Read from versions.yaml
+	repo=$(get_from_kata_deps "externals.coco-kbs.url")
+	version=$(get_from_kata_deps "externals.coco-kbs.version")
+	image=$(get_from_kata_deps "externals.coco-kbs.image")
+	image_tag=$(get_from_kata_deps "externals.coco-kbs.image_tag")
+
+	if [ -d "$COCO_KBS_DIR" ]; then
+		rm -rf "$COCO_KBS_DIR"
+	fi
```

Comment:
Personal preference, but you could collapse this to a single line:

```suggestion
	[ -d "$COCO_KBS_DIR" ] && rm -rf "$COCO_KBS_DIR"
```

Reply:
maybe also add a trap to see that is deleted